src/py42/exceptions.py

```diff
@@ -167,6 +167,15 @@ def __init__(self, exception, message=None):
         super(Py42UserAlreadyExistsError, self).__init__(exception, message)
 
 
+class Py42CloudAliasLimitExceeded(Py42BadRequestError):
+    """An exception raised when number of cloud aliases exceeds against max allowed
```

Comment:
The doc does not describe enough about _when_ it is actually raised.

Perhaps something like `An Exception raised when trying to add a cloud alias to a user when that user already has the max amount of supported cloud aliases`.

Note the missing word `the` between `when` and `number`,

Also note that this more accurately tells users to expect this exception during some type of `add` operation (adding aliases).

Reply:
Changed.